openvdb/openvdb/tools/VolumeToMesh.h

```diff
@@ -59,6 +59,14 @@ volumeToMesh(
 
 
 /// @brief Adaptively mesh any scalar grid that has a continuous isosurface.
+/// @details When converting to polygons, the adaptivity threshold determines
+///   how closely the isosurface is matched by the resulting mesh. Higher
+///   thresholds will allow more variation in polygon size, using fewer
+///   polygons to express the surface. Triangles will only be created for
+///   areas of the mesh which hit the adaptivity threshold and can't be
+///   represented as quads.
+/// @note Do not use this method just to get a triangle mesh - use the above
+///   method and post process the quad index list.
```

Comment:
Should we add something along the line of:
"The algorithm is based on the method Dual Contouring of Hermite Data by Ju et al. and its implementation for sparse grid is documented in Alden's thesis' "Fast and Adaptive Polygon Conversion By Means Of Sparse Volumes"."

Reply:
Going to merge as is as it addressed the specific linked issue but please feel free to add and expand this - the top doxygen to this file could really use more description